conan/cli/commands/cache.py

```diff
@@ -18,55 +18,37 @@ def cache_path(conan_api: ConanAPIV2, parser, subparser, *args):
         Shows the path af a given reference
     """
     subparser.add_argument("reference", help="Recipe reference or Package reference")
-    subparser.add_argument("--folder", choices=['exports', 'exports_sources', 'sources', 'build',
-                                                'package'], default="exports",
```

Comment:
Why removing the `--folder=package`?

Reply:
Because it is redundant, not needed. By default it is giving you the ``package`` folder (95% of cases for a full ``pref``). Why adding another argument that will give further incompatibilities when the arg is a ``ref``. 
Same with ``export`` folder, that is the default for a ``ref``